tests/test_routes.py

```diff
@@ -349,4 +383,4 @@ def test_get_item_order_not_found(self):
         It should not Read an item when the order is not found
         """
         resp = self.app.get(f"{BASE_URL}/0/items/0")
-        self.assertEqual(resp.status_code, status.HTTP_404_NOT_FOUND)
+        self.assertEqual(resp.status_code, status.HTTP_404_NOT_FOUND)
```

Comment:
Please don't remove new line from the end of file. Run `make lint` command to make sure that your code doesn't introduce any linting issues.